com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -3,7 +3,8 @@
 using UnityEngine;
 using System.Reflection;
 using System.Linq;
-using System.IO;
+using Unity.Collections;
+using Unity.Netcode.Messages;
```

Comment:
we should use top-level `Unity.Netcode` namespace and not subset it within the same asmdef. (see #1009)

Reply:
Done. (Stupid Rider automatically setting the namespace...)